app/src/main/res/values/strings.xml

```diff
@@ -58,4 +58,7 @@
     <string name="lyrics">Lyrics</string>
     <string name="lyrics_action_save">Save</string>
     <string name="lyrics_action_delete">Delete</string>
+    <!-- Internet connection -->
+    <string name="internet_connection_absent">Something went wrong…</string>
+    <string name="internet_connection_retry">Retry</string>
```

Comment:
Naming is important: so instead of internet_connection_absent you can make some normal name